pkg/util/azureclient/typeToCode.go

```diff
@@ -0,0 +1,32 @@
+package azureclient
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import "strings"
+
+// GetAPIVersionFromType gets the APIVersion from a full resource type
+func GetAPIVersionFromType(_type string) string {
+	defaultVer := APIVersions["Microsoft.Compute"]
```

Comment:
why is this needed and how can it help?

Reply:
well what are the other options? return an empty string/error 
I was hoping that at least it would glean some info - better than nothing

Reply:
if we have an unknown resource type, there is no guarantee that the compute api version will work for it.  We should find a different mechanism to indicate this fault.  A log line + leaving the item unenriched should be OK, shouldn't it?